geometry_msgs/Point startXYZ
geometry_msgs/Point goalXYZ
---
geometry_msgs/Point[] pathSrv